packages/axios-extension/src/abap/app-index-service.ts

```diff
@@ -34,4 +34,28 @@ export abstract class AppIndexService extends Axios implements Service {
         const response = await this.get('/', { params });
         return JSON.parse(response.data).results as AppIndex;
     }
+
+    /**
+     * Returns if manifest is first supported.
```

Comment:
I don't understand what "manifest is first" means. Any chance you could describe it differently.

Reply:
Fixed